tests/test.py

```diff
@@ -1056,6 +1092,11 @@ def get_connector_api_connection_string(self, port=None):
             port = self.CONNECTOR_API_PORT_1
         return get_connector_connection_string(port)
 
+    def get_ocsp_server_connection_string(self, port=None):
+        if not port:
+            port = self.OCSPSERVER_PORT
+        return 'http://127.0.0.1:{}'.format(port)
```

Comment:
I can imagine a (perverse) desire to run tests with a real OCSP server running elsewhere, so I guess it wouldn't hurt to make the hostname configurable too.